tools/depends/target/ffmpeg/autobuild.sh

```diff
@@ -163,7 +163,6 @@ CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS" LDFLAGS="$LDFLAGS" \
 	--disable-mips32r2 \
 	--disable-mipsdspr1 \
 	--disable-mipsdspr2 \
```

Comment:
according to travis, these options seem to have been removed
Unknown option "--disable-mipsdspr1".